src/main/webapp/scripts/ou-utils.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Currently just returns the first OU matching the name search criterion.
+*/
+function searchOU(searchName) {
```

Comment:
can this be replaced by a for(x of y) loop

Reply:
done